builtin/logical/ssh/path_roles.go

```diff
@@ -336,6 +511,23 @@ func (b *backend) pathRoleRead(req *logical.Request, d *framework.FieldData) (*l
 				"allowed_users":     role.AllowedUsers,
 			},
 		}, nil
+	} else if role.KeyType == KeyTypeCA {
+		return &logical.Response{
+			Data: map[string]interface{}{
+				"allowed_users":            role.AllowedUsers,
+				"allowed_domains":          role.AllowedDomains,
+				"default_user":             role.DefaultUser,
+				"max_ttl":                  role.MaxTTL,
+				"ttl":                      role.TTL,
+				"allowed_critical_options": role.AllowedCriticalOptions,
```

Comment:
Are `default_critical_options` and `default_extensions` not here for a reason or were they skipped by mistake?